http/oidc/src/test/java/org/wildfly/security/http/oidc/OidcTest.java

```diff
@@ -291,4 +353,18 @@ private InputStream getOidcConfigurationInputStreamWithTokenSignatureAlgorithm()
                 "}";
         return new ByteArrayInputStream(oidcConfig.getBytes(StandardCharsets.UTF_8));
     }
+
+    private InputStream getOidcConfigurationInputStreamWithScope(String clientSecret, String scopeValue){
```

Comment:
I think clientSecret is unused here right since CLIENT_SECRET is used instead?

Reply:
removed